api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -56,43 +56,10 @@
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
+import org.openmrs.*;
 import org.openmrs.Order.Action;
```

Comment:
you should add the preferred openmrsformatter to your IDE to avoid these * imports and unwanted formatting changes.